src/features/Courses/AddClass/index.jsx

```diff
@@ -20,7 +21,7 @@ import { initialPage } from 'features/constants';
 import 'features/Courses/AddClass/index.scss';
 
 const AddClass = ({
-  isOpen, onClose, courseInfo, isCoursePage,
+  isOpen, onClose, courseInfo, isCoursePage, isEdit,
```

Comment:
`isEditing` instead of `isEdit` can provide clarity and improve readability 

```suggestion
  isOpen, onClose, courseInfo, isCoursePage, isEditing,
```